packages/next/server/body-streams.ts

```diff
@@ -0,0 +1,87 @@
+import type { IncomingMessage } from 'http'
+import { Readable } from 'stream'
+import { TransformStream } from 'next/dist/compiled/web-streams-polyfill/ponyfill'
```

Comment:
The pre-compiled web-streams-polyfill is using ponyfill
```suggestion
import { TransformStream } from 'next/dist/compiled/web-streams-polyfill'
```

Reply:
done! 👍